package.json

```diff
@@ -38,7 +38,8 @@
     "is-valid-path": "^0.1.1",
     "node-fetch": "^2.6.1",
     "recursive-readdir": "^2.2.2",
-    "sharp": "^0.27.0"
+    "sharp": "^0.27.0",
+    "shorthash2": "^1.0.3"
```

Comment:
Seems a cool and small library but i'm worried about maintenance due to low amount of usage. It would be probably safer using hasha since we are in NodeJS namespace. (if finally use hash. please see other comments)